app/src/main/java/com/google/maps/android/compose/LocationTrackingActivity.kt

```diff
@@ -100,6 +115,7 @@ class LocationTrackingActivity : AppCompatActivity() {
                     onMapLoaded = {
                         isMapLoaded = true
                     },
+                    onMyLocationButtonClick = {  Log.d(TAG,"Overriding the onMyLocationButtonClick with this Log"); true },
```

Comment:
This line could benefit from a comment indicating that it's demonstrating use of a Custom My Location Button. Otherwise people might end up leaving it empty and unwittingly disable the default My Location Button.